src/KeyboardLayouts/ImprovedKeylayouts.h

```diff
@@ -24,11 +24,59 @@ THE SOFTWARE.
 // Include guard
 #pragma once
 
-#if !defined(LAYOUT_US_ENGLISH)
-#error This API does not support non US english layouts.
+// Keyboard Modifiers
+enum KeyboardMods : uint16_t {
+	MOD_LEFT_CTRL		= 0b0000000100000000,
+	MOD_LEFT_SHIFT		= 0b0000001000000000,
+	MOD_LEFT_ALT		= 0b0000010000000000,
+	MOD_LEFT_GUI		= 0b0000100000000000,
+	MOD_RIGHT_CTRL		= 0b0001000000000000,
+	MOD_RIGHT_SHIFT		= 0b0010000000000000,
+	MOD_RIGHT_ALT		= 0b0100000000000000,
+	MOD_RIGHT_GUI		= 0b1000000000000000,
+};
+
+// Keyboard Leds
+enum KeyboardLeds : uint8_t {
+	LED_NUM_LOCK		= (1 << 0),
+	LED_CAPS_LOCK		= (1 << 1),
+	LED_SCROLL_LOCK		= (1 << 2),
+	LED_COMPOSE			= (1 << 3),
+	LED_KANA			= (1 << 4),
+	LED_POWER			= (1 << 5),
+	LED_SHIFT			= (1 << 6),
+	LED_DO_NOT_DISTURB	= (1 << 7),
+};
+
+#if defined(LAYOUT_US_ENGLISH)
+	//#include "ImprovedKeylayoutsUS.h"
```

Comment:
Why is this commented? Does US layout still work?

Reply:
I commented it out while debugging and then forgot about it.
US layout now works as expected.